.github/workflows/atlas.yml

```diff
@@ -15,44 +15,29 @@ jobs:
     runs-on: ubuntu-latest
```

Comment:
please revert all ci file changes, they are out of scope to this pr.

Reply:
ohhhhh 

you don't know:

those cause it to work.

Reply:
CI changes ought to be restricted to removing the diff action. Did you see anything more than that?

Without removing the diff action, this PR literally truly kills ci.

To my surprise, after taking them out it worked without further modification.